src/core2rust.rkt

```diff
@@ -0,0 +1,113 @@
+#lang racket
+
+(require "imperative.rkt")
+
+(provide rust-header core->rust rust-supported)
+
+(define rust-header (lambda (_) "#![allow(non_snake_case, unused_mut, unused_parens)]\n\n"))
```

Comment:
You might be able to force names into snake case by passing in a procedure for the `#:fix-name` keyword for `make-imperative-compiler` (see `fix-name` in `src/imperative.rkt`; this is the default implementation for imperative compilers).